wormhole-connect/src/hooks/useAvailableRoutes.ts

```diff
@@ -28,54 +26,51 @@ const useAvailableRoutes = (): void => {
     let isActive = true;
 
     const getAvailable = async () => {
-      const routes: RouteState[] = [];
-      for (const value of config.routes) {
-        const r = value as Route;
-
+      let routes: RouteState[] = [];
+      await config.routes.forEach(async (name, route) => {
```

Comment:
I think we should use `Promise.all(routes.map(...))` or a `for ... of` loop to avoid issues calling async functions in `forEach`: https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971#do-not-use-foreach-with-async-await since